chat-client/src/client/mynahUi.ts

```diff
@@ -237,14 +244,30 @@ export const createMynahUi = (messager: Messager, tabFactory: TabFactory): [Myna
             }
             messager.onInfoLinkClick(payload)
         },
+        onInBodyButtonClicked: (tabId, messageId, action, eventId) => {
+            if (action.id === disclaimerAcknowledgeButtonId) {
+                // Hide the legal disclaimer card
+                disclaimerCardActive = false
+
+                // Update the disclaimer getting acknowledged
+                messager.onDisclaimerAcknowledged()
+
+                // TODO: create telemetry
+
+                // Remove all disclaimer cards from all tabs
+                Object.keys(mynahUi.getAllTabs()).forEach(storeTabKey => {
+                    mynahUi.updateStore(storeTabKey, { promptInputStickyCard: null })
+                })
+            }
+        },
         tabs: {
             [initialTabId]: {
                 isSelected: true,
-                store: tabFactory.createTab(true),
+                store: tabFactory.createTab(true, disclaimerCardActive),
             },
         },
         defaults: {
-            store: tabFactory.createTab(true),
+            store: tabFactory.createTab(true, disclaimerCardActive),
```

Comment:
We should not give it on defaults. It should only work when a new tab is open by checking the acknowledgement state and add it depending on the situation. 

When it is acknowledged, it will not come back. So what we need to do is to check only `should we add it, or not.`. But if we add it to defaults, it turns out to `should we REMOVE it or not`. We should not care about the removal part, only adding depending on the statement should be fine.

We don't need it in defaults.